libs/partners/voyageai/pyproject.toml

```diff
@@ -19,9 +19,9 @@ disallow_untyped_defs = "True"
 "Release Notes" = "https://github.com/langchain-ai/langchain/releases?q=tag%3A%22langchain-voyageai%3D%3D0%22&expanded=true"
 
 [tool.poetry.dependencies]
-python = ">=3.9,<4.0"
+python = ">=3.9,<3.13"
```

Comment:
What is the status of python 3.13 support before and after this PR?

Reply:
About a month ago, when they moved to `0.3.0` they have changed the dependency from: `^3.7.1` to: `^3.9,<3.13`, but here the requirement where: `>=3.9,<4.0`.
So I wonder how can I approach that, as otherwise it seems we cannot use any version above `0.3.*` version.